app/styles/modules/_custom-rows.scss

```diff
@@ -162,6 +162,24 @@ ul.links {
   margin: 0 10px;
 }
 
+.show-password-label {
+  background-color: $show-password-background-color;
+  color: $show-password-label-color;
+  -webkit-transition: background-color 450ms,
+                      color 450ms;
+  transition: background-color 450ms,
+              color 450ms;
+}
+
+.password:focus ~ .show-password-label {
+  color: $input-row-focus-border-color;
+}
+
+.password[type=text]:focus ~ .show-password-label {
```

Comment:
I'd say stick with .password, seeing as we don't have .show-password-label after other inputs. I think it'd be better to be as specific as possible.